js/noms/src/http-batch-store-test.js

```diff
@@ -9,11 +9,13 @@ import {assert} from 'chai';
 import Hash from './hash.js';
 import {emptyHash} from './hash.js';
 import HttpBatchStore from './http-batch-store.js';
-import mock from 'mock-require';
+import {invariant} from './assert.js';
+
+declare var jest;
```

Comment:
Is it possible to make `jest` be default-exported from jest.js?

(obviously it would be fake, in a sense, but better for documentation)